SpatialConvolutionMM.lua

```diff
@@ -70,19 +70,49 @@ function SpatialConvolutionMM:updateOutput(input)
       self.padding = nil
    end
    input = makeContiguous(self, input)
-   return input.nn.SpatialConvolutionMM_updateOutput(self, input)
+   input.THNN.SpatialConvolutionMM_updateOutput(
+      input:cdata(),
+      self.output:cdata(),
+      self.weight:cdata(),
+      self.bias:cdata(),
+      self.finput:cdata(),
+      self.kW, self.kH,
+      self.dW, self.dH,
+      self.padW, self.padH
+   )
```

Comment:
I'm thinking about adding a loop in lua side iterating, like in cudnn. Do you think we need to do it in C (to make better use of streams in cuda, etc) ?